tests/test_bindings.py

```diff
@@ -1,33 +1,47 @@
+'''TESTS BINDING'''
+
+import os
 import json
 import yaml
-import vttv
-
 import sys
-import os
+
+import vttv
 
 # source dir is the directory a level above this file
 source_dir = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
+# Read the YAML config file
 with open(f'{source_dir}/tests/test_bindings_conf.yaml', 'r') as stream:
-  try:
-    params = yaml.safe_load(stream)
-  except yaml.YAMLError as exc:
-    print(exc)
+    try:
+        params = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+# Check main key is "visualization"
+if 'visualization' not in params:
```

Comment:
Code style for Python in LBAF ecosystem is to conform to C-style simple quotes for single characters, vs. double quotes for string literals. I would suggest respecting the same style here for consistency.